.github/actions/deploy-environment/action.yml

```diff
@@ -58,7 +58,7 @@ runs:
     shell: bash
     working-directory: cluster
 
-  - uses: Azure/login@v1
+  - uses: Azure/login@v1.6.1
```

Comment:
Does this make a difference? v1 should download the latest v1.x.x version

Reply:
Fixed